sql-cli/conftest.py

```diff
@@ -147,6 +154,38 @@ def initialised_project_with_test_config(initialised_project: Project):
     return initialised_project
 
 
+@pytest.fixture()
+def initialised_project_with_load_file_workflow(
+    root_directory_templates_load_file, initialised_project: Project
+):
+    output_table_name = create_unique_table_name(UNIQUE_HASH_SIZE)
+    render_kwargs = {
+        "context": {"output_table_name": output_table_name},
+        "searchpath": root_directory_templates_load_file,
+    }
+
+    render(
+        template_file=Path("load_homes_main.yaml.jinja2"),
+        output_file=initialised_project.directory
+        / "workflows"
+        / "example_load_file"
+        / f"{output_table_name}.yaml",
+        **render_kwargs,
+    )
+    os.remove(initialised_project.directory / "workflows" / "example_load_file" / "load_homes_main.yaml")
+
+    render(
+        template_file=Path("transform_homes_main.sql.jinja2"),
+        output_file=initialised_project.directory
+        / "workflows"
+        / "example_load_file"
+        / "transform_homes_main.sql",
+        **render_kwargs,
+    )
+
+    return initialised_project
```

Comment:
It feels that it would be worth having an example workflow with the load. If we had that, would it still make sense to render it in this fixture?

Reply:
Yes, because what we do in that fixture is that we dynamically change the output table name as when running it concurrently the jobs can interfere on the same table.

Reply:
We do have an example workflow with the load that gets added on project initialisation. We added this fixture as tests running concurrently across Airflow version were creating the same output table and were conflicting across different runs of Airflow version. Here, we manipulate the created workflow to have output table with unique name so that concurrent runs succeed well.